### PR TITLE
Update and add NPC movement scripts - STV

### PR DESCRIPTION
**Pull request by NickTyrer**
This PR adds some new movement scripts and updates existing ones using waypoints from sniffs where possible.

### CODE REVIEW

File: sql/migrations/20200706205532_world.sql [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+(754, 15, -11378.7, -226.508, 73.5829, 100, 0, 0, 0),
+(754, 16, -11375.4, -219.386, 75.0807, 100, 0, 0, 0),
+(754, 17, -11373, -216.171, 75.2465, 100, 0, 0, 0),
+(754, 18, -11365.7, -212.828, 75.2215, 100, 0, 0, 0),
+(754, 19, -11353.7, -211.491, 75.2339, 100, 0, 0, 0),
+(754, 20, -11345.6, -211.416, 75.2236, 100, 0, 0, 0),
+(754, 21, -11337.5, -210.781, 75.2219, 100, 0, 0, 0),
+(754, 22, -11333.4, -210.355, 75.283, 100, 0, 0, 0);
+
+-- add private merle waypoints
```
Comment (alexkulya) [inline sql/migrations/20200706205532_world.sql:90]: Are you sure that this NPC have OOC movement?
https://github.com/vmangos/core/commit/e283993180f7fe4b7601bf265f8ab7d6f8159a5f

Comment (NickTyrer) [reply sql/migrations/20200706205532_world.sql:90]: He just patrols the camp constantly, just checked classic.

Comment (NickTyrer) [reply sql/migrations/20200706205532_world.sql:90]: He does have a script where he runs out of the camp with the rebel watchman chasing him. Not sure what that is.

**Comment by NickTyrer** [comment]
TODO: Private thorsen script not completely working: 
 
- Add despawn animation to gameobject 12165.

**Comment by ratkosrb** [comment]
Drop Thorsen from the PR and I'll implement him later.

File: sql/migrations/20200706205532_world.sql [hunk 2]
```diff
@@ -0,0 +1,1148 @@
...
+(610, 11, -12383.7, 203.173, 1.99872, 100, 0, 0, 0),
+(610, 12, -12383.4, 206.133, 2.02113, 100, 0, 0, 0),
+(610, 13, -12383.3, 215.205, 2.3636, 100, 0, 0, 0),
+(610, 14, -12384.2, 224.195, 2.64434, 100, 0, 0, 0);
+
+-- commander aggro gosh should roam
+UPDATE `creature` SET `movement_type` = 1, `wander_distance` = 3 WHERE `guid` = 721;
+
+-- remove grunt
+DELETE FROM `creature` WHERE `guid` = 610;
```
Comment (ratkosrb) [inline sql/migrations/20200706205532_world.sql]: You add waypoints for this guid above, but now you delete him? This produces a startup error about unused waypoints for guid 610.

Comment (NickTyrer) [reply sql/migrations/20200706205532_world.sql]: Corrected

**Pull request merged**